step/serialization.md

```diff
@@ -523,31 +512,35 @@ Exception in thread "main" java.io.InvalidClassException: filter status: REJECTE
 	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1689)
 	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:495)
 	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:453)
-	at ClassSerializationServer.main(ClassSerializationServer.java:27)
+	at SerializationServer.main(SerializationServer.java:27)
 ```
 
-**Extra Credit**: You can further confirm that the filter will only reject `Runnable` by updating `ClassSerializationClient` to send `MessageClass` and see that the code executes successfully. 
+**Extra Credit**: You can further confirm that the filter will only reject `Runnable` by updating `SerializationClient` to send `Message` and see that the code executes successfully. 
```

Comment:
PLease remove

Reply:
Can you add some quick example using the command-line, ex. set max-size, limit depth-level?